src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceConditionService.kt

```diff
@@ -186,6 +321,23 @@ class LicenceConditionService(
         BespokeCondition(licence = licenceEntity, conditionSequence = index, conditionText = condition)
       )
     }
+
+    val changes = mapOf(
+      "typeOfChange" to "update",
+      "condition" to "bespoke",
+      "changes" to request.conditions
+    )
```

Comment:
Am I correct in thinking that this won't show a before-and-after of what the changes to bespoke conditions are?

Reply:
Functionality added